src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -85,3 +85,21 @@ def register_submission(
 
     def get_reference_from_result(self, result: None) -> NoReturn:
         raise NoSubmissionReference("Object API plugin does not emit a reference")
+
+    def test_config():
+        test = []
+        config = ObjectsAPIConfig.get_solo()
+        drc_client = config.drc_service.build_client()
+        orc_client = config.objects_service.build_client()
+
+        clients = [{'name': 'ORC (Overige)', 'type': 'object', 'client': orc_client},
+                   {'name': 'DRC (Informatieobjecten)', 'type': 'document', 'client': drc_client},
+                ]
```

Comment:
You're duplicating the name that is also in the service model choices. No need to duplicate strings here.